packages/beacon-node/src/sync/range/utils/hashBlocks.ts

```diff
@@ -16,7 +16,7 @@ export function hashBlocks(blocks: BlockInput[], config: ChainForkConfig): RootH
     }
     default: {
       const block0 = blocks[0].block;
-      const blockN = blocks[blocks.length - 1].block;
+      const blockN = blocks.at(-1)?.block as BlockInput["block"];
```

Comment:
Same question on this? Aren't we manually defeating the purpose of the `at` method here?